src/Grok.Net.Tests/UnitTests.cs

```diff
@@ -301,5 +302,40 @@ public void Parse_Base64String_Patterns(string base64String)
             Assert.Equal(base64String, grokResult[0].Value);
             Assert.Single(grokResult);
         }
+
+        [Fact]
+        public void Parse_Multiline()
+        {
+            // Arrange
+            const string timeKeyword = "loggingTime";
+            const string messageKeyword = "message";
+            const RegexOptions options = RegexOptions.Singleline;
+
+            var defaultGrok = new Grok($"%{{TIMESTAMP_ISO8601:{timeKeyword}}} %{{GREEDYDATA:{messageKeyword}}}");
```

Comment:
Do we need to check the "default" in the Multiline test?

Reply:
Not really, this is for comparison. Could be removed if you want a more concise unit test.